tensorflow/python/keras/engine/saving.py

```diff
@@ -73,8 +68,10 @@ def save_model(model, filepath, overwrite=True, include_optimizer=True):
   Raises:
       ImportError: if h5py is not available.
   """
-
-  if h5py is None:
+  # pylint: disable=g-import-not-at-top
```

Comment:
@aylusltd Sorry for the delay in reviewing. There is duplication below. Can you refactor the following few lines into a module-level function and call the function in both places? Also: the pylint disable line should be applied to only the `import h5py` line, so that the linter item is not disabled throughout the .py file.